shell/models/management.cattle.io.kontainerdriver.js

```diff
@@ -18,6 +18,7 @@ export const KONTAINER_TO_DRIVER = {
   lkeengine:                        'linodelke',
   okeengine:                        'oracleoke',
   oraclecontainerengine:            'oracleoke', // Does this actually exist?
+  ociocneengine:                    'ociocne',
```

Comment:
Changes under shell need to be demarcated with the following marker comments:
// Added by Verrazzano Start
**** ... code changes ... ****
// Added by Verrazzano End